.github/workflows/builder.yml

```diff
@@ -111,6 +111,20 @@ jobs:
         run: |
           rm -rf /tmp/.buildx-cache
           mv /tmp/.buildx-cache-new /tmp/.buildx-cache
+      - name: Send WASM to Discord
+        uses: tsickert/discord-webhook@v4.0.0
+        with:
+          webhook-url: ${{ secrets.DISCORD_WEBHOOK }}
+          username: "DevOps Bot"
+          avatar-url: ${{ secrets.LOGO_URL }}
+          embed-title: New build wasm
+          content: "<@839002855896907787> <@813609814650388491> <@739126196469497989>"
+          embed-description: "Build substrate success! Download this new wasm file."
```

Comment:
We need build information, at least commit hash, maybe like this
"Build success on commit sha `cHe657`"